distribution/docs/src/main/resources/_contents/_catalog-contents/extending-catalog-contents.adoc

```diff
@@ -1255,7 +1255,8 @@ This plugin can be installed and uninstalled using the normal processes describe
 No configuration is necessary for this plugin. 
```

Comment:
I think it's still accurate. It's preconfigured to default values so no configuration is required, but it can be configured by the admin to get a different behavior. That's how I understood it at least.